ed448/src/curve/edwards/affine.rs

```diff
@@ -46,7 +46,9 @@ impl elliptic_curve::point::AffineCoordinates for AffinePoint {
     type FieldRepr = Ed448FieldBytes;
 
     fn x(&self) -> Self::FieldRepr {
-        Ed448FieldBytes::clone_from_slice(&self.x.to_bytes())
+        todo!()
+        // size missmatch
+        // Ed448FieldBytes::from(self.x.to_bytes())
```

Comment:
@mikelodder7 @tarcieri I found a couple of these, I am not sure if this is a bug in the original implementation, or I am missing sth in the translation

Reply:
That was me that implemented it. The trait doesn't really align with edwards curves given they store the `y` and not the `x`. This is kludgey for sure.